test/Microsoft.Azure.ServiceBus.UnitTests/BrokeredMessageTests.cs

```diff
@@ -19,7 +19,11 @@ public class BrokeredMessageTests
                 ReceiveMode.ReceiveAndDelete);
 
             await TestUtility.SendMessagesAsync(queueClient.InnerSender, 1);
-            var message = await queueClient.ReceiveAsync();
+            var message = await queueClient.PeekAsync();
```

Comment:
This is awesome. Really. Functionality backed up by a test.
But can I ask a favour (as a payback for finding the bug)? Could you please (nice please) extract it into a method that would be self contained and disclose what it does? _Peeked_message_should_not_be_allowed_to_be_completed_ as an example. Bam! Right there if it fails, I know what it does 😃